core/workspace_svg.js

```diff
@@ -1011,6 +1011,14 @@ Blockly.WorkspaceSvg.prototype.isDragging = function() {
 };
 
 /**
+ * Is this workspace draggable and scrollable?
+ * @return {boolean} True if this workspace may be dragged.
+ */
+Blockly.WorkspaceSvg.prototype.isDraggable = function() {
```

Comment:
Do we want this in the public api? Seems harmless though I can't think of what an external developer might do with it... but just making sure we're aware.